### PR TITLE
lestarch: fixing graalvm build

### PR DESCRIPTION
**Pull request by LeStarch**
@bocchino @tiffany1618 this should fix the graal VM build.

### CODE REVIEW

**Comment by LeStarch** [comment]
Also, those warnings are from the fact that all classes are not used in all native images.  Thus they really do not exist, and the warning are innocuous.

**Comment by LeStarch** [comment]
Once merged, we should build an `a3` release and bump the F´  version to test CI.

File: compiler/release [hunk 1]
```diff
@@ -76,6 +76,15 @@ sync; sync; sync;
 rm "$native_bin"/*.txt
 sync; sync; sync;
 
+# Testing of the native binaries
+mv "$directory/bin" "$directory/bin.bak"
```
Comment (bocchino) [inline compiler/release:80]: Let's add `rm -rf bin.bak` here. Otherwise the build will fail if bin.bak exists already.

**Review by bocchino — COMMENTED** [review]
Looks good! Just one comment.

**Comment by LeStarch** [comment]
@bocchino I just changed the move to a `rm`.  Since release runs primarily in CI, there isn't a lot driving us to keep these files around.

**Review by bocchino — APPROVED** [review]
Looks good!

**Pull request merged**